runners/spark/spark_runner.gradle

```diff
@@ -163,12 +163,12 @@ dependencies {
   implementation library.java.vendored_grpc_1_36_0
   implementation library.java.vendored_guava_26_0_jre
   implementation "com.codahale.metrics:metrics-core:3.0.1"
-  compileOnly "org.apache.spark:spark-core_$spark_scala_version:$spark_version"
-  compileOnly "org.apache.spark:spark-network-common_$spark_scala_version:$spark_version"
+  implementation "org.apache.spark:spark-core_$spark_scala_version:$spark_version"
```

Comment:
We should also remove these from the `testImplementation` below since they are redundant now that they are in `implementation`.

Reply:
Done.